articles/guides/ip-whitelist.md

```diff
@@ -0,0 +1,41 @@
+## Whitelist IP Addresses
```

Comment:
Since you've created a new doc, this now should be a heading 1. (And all other headings on the page should move up a level.)